clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -66,7 +67,8 @@ async function isJavaInstalled() {
     return new Promise<any>((resolve, reject) => {
         const ls = cp.spawn("java", ["-version"]);
         ls.stderr.on("data", (data: any) => {
-            if (!data.toString().includes('java version "1.8')) {
+            var javaCheck = new DefaultJavaVersionCheck();
```

Comment:
MS recommends to avoid var: ```With TypeScript being a superset of JavaScript, the language naturally supports let and const. Here we’ll elaborate more on these new declarations and why they’re preferable to var.```
https://www.typescriptlang.org/docs/handbook/variable-declarations.html